maquetado/scss/login.scss

```diff
@@ -0,0 +1,94 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wolox-image-login {
+  align-self: center;
+  height: 70px;
+  margin-top: 20px;
+  width: 200px;
+}
+
+.login-container {
+  align-items: center;
+  background-color: $light-grey;
+  border-top: 10px solid $light-blue;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: flex-start;
```

Comment:
If I am not mistaken you don't need to set this. By default it is always set as flex-start